plugins/mimecast/komand_mimecast/tasks/monitor_siem_logs/task.py

```diff
@@ -45,11 +45,11 @@ def run(self, params={}, state={}) -> (List[Dict], Dict):  # pylint: disable=unu
                         break
                 except ApiClientException as error:
                     self.logger.error(
+                        f"returning state={state}, has_more_pages={has_more_pages}"
```

Comment:
This line is already part of this log line on L50.

Reply:
my bad in last commit removing an old logger I copied over incorrectly will remove